extension-impl/remoting-http/src/main/java/com/alipay/sofa/rpc/transport/http/Http2ClientInitializer.java

```diff
@@ -156,4 +169,29 @@ public void userEventTriggered(ChannelHandlerContext ctx, Object evt) throws Exc
             ctx.fireUserEventTriggered(evt);
         }
     }
+
+    /**
+     * Configure the pipeline for a cleartext useing Prior-Knowledge method to start http2.
+     */
+    private void configureClearTextPriorKnowledge(SocketChannel ch) {
+        ch.pipeline().addLast(connectionHandler,
+            new PrefaceFrameWrittenEventHandler(),
+            new UserEventLogger());
+        configureEndOfPipeline(ch.pipeline());
+    }
+
+    /**
+     * Class that capture Prior-Knowledge method's Preface frame written Events triggered on this channel,
+     * and Flush Channel for send preface frame to remote realy
+     */
+    private static class PrefaceFrameWrittenEventHandler extends ChannelInboundHandlerAdapter {
+        @Override
+        public void userEventTriggered(ChannelHandlerContext ctx, Object evt) throws Exception {
+            System.out.println("User Event Handler :" + evt.toString());
```

Comment:
Use `LOGGER` instead of `System.out.println`

Reply:
ah, it is learned from previous code's UserEventLogger, I will change it